.env

```
@@ -0,0 +1 @@
+API = "https://flower.elevateegy.com/api/v1"
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)